<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0.004" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2017-08-01T12:46:27" tests="0" failures="0" time="0">
  </testsuite>
  <testsuite name="tests for account model" timestamp="2017-08-01T12:46:27" tests="1" file="/Users/lwo10/skykids/test/model/accountsTest.js" failures="1" time="0.004">
    <testcase name="tests for account model account exists" time="0.004" classname="account exists">
      <failure><![CDATA[AssertionError: expected 2 to equal 5
    at Context.<anonymous> (test/model/accountsTest.js:14:18)
    at Context.<anonymous> (test/model/accountsTest.js:10:9)]]></failure>
    </testcase>
  </testsuite>
</testsuites>